app/src/main/java/com/kickstarter/ui/activities/ProjectUpdatesActivity.java

```diff
@@ -85,6 +97,11 @@ private boolean handleProjectUpdateUriRequest(final @NonNull Request request, fi
     return true;
   }
 
+  private boolean handleProjectUpdatesUriRequest(final @NonNull Request request, final @NonNull WebView webView) {
+    this.viewModel.inputs.goToUpdatesRequest(request);
+    return false;
```

Comment:
this handler, as well as [`handleProjectUpdateUriRequest` in UpdateViewModel](https://github.com/kickstarter/android-oss/pull/61/files#diff-e5291b7fe239077b3e0a95cab8dd7691R114), return `false` rather than true because we don't actually want the web view to handle the request--we just want to use this method as a callback to let our view model know that the relevant url request has been intercepted